ansible/roles/mysql-5.7-original/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: specify MySQL root password before installing
```

Comment:
@scuben dooing it this way instead of how i did it, does not allow to update the user and password on reprovision